javascript/bower.json

```diff
@@ -0,0 +1,45 @@
+{
+  "name": "OLA",
+  "version": "9.4.0",
```

Comment:
This should be 0.9.4, but it really needs to be generated by make. See here for an example of how to do that:
https://github.com/OpenLightingProject/ola/commit/f8af6c3b29f2d236b63970c19a9ad02d1e9603ef